locale/en/download/package-manager.md

```diff
@@ -36,7 +36,7 @@ pacman -S nodejs npm
 
 Also including: **Linux Mint**, **Linux Mint Debian Edition (LMDE)**, **elementaryOS** and others.
 
-Node.js is available from the [NodeSource](https://nodesource.com) Debian and Ubuntu binary distributions repository (formerly [Chris Lea's](https://github.com/chrislea) Launchpad PPA). Support for this repository, along with its scripts, can be found on GitHub at [nodesource/distributions](https://github.com/nodesource/distributions).
+Node.js is available from the [NodeSource](http://nsrc.io/release-updates-2) Debian and Ubuntu binary distributions repository (formerly [Chris Lea's](https://github.com/chrislea) Launchpad PPA). Support for this repository, along with its scripts, can be found on GitHub at [nodesource/distributions](https://github.com/nodesource/distributions).
```

Comment:
How about if we keep the existing link to nodesource and add after that you can sign up to get immediate updates when new versions come out? Does that work?